src/arch/armv8/gicv3.c

```diff
@@ -27,6 +27,11 @@ size_t gich_num_lrs()
 
 static inline void gicc_init()
 {
+    size_t cpu_has_gicv = sysreg_id_aa64pfr0_el1_read() & ID_AA64PFR0_GIC_MSK;
+    if (!cpu_has_gicv) {
```

Comment:
Just catch this. Shouldn't this variable be called `cpu_has_gic_sysreg` or something similar? Just because in the GIC spec terminology, GICV refers to the GIC virtual CPU interface...